guides/common/modules/proc_synchronizing-a-single-repository.adoc

```diff
@@ -20,7 +20,7 @@ ifdef::client-content-dnf[]
 For more information, see xref:Enabling_Red_Hat_Repositories_{context}[].
 endif::[]
 +
-If you want to sync custom content, first xref:Creating_a_Custom_Product_{context}[create a custom Product] and xref:Synchronizing_Repositories_{context}[synchronize Product repositories].
+If you want to sync custom content, first xref:Creating_a_Custom_Product_{context}[create a custom product] and xref:Synchronizing_Repositories_{context}[synchronize product repositories].
```

Comment:
unrelated: "product repositories" feels weird to me:

```suggestion
If you want to sync custom content, first xref:Creating_a_Custom_Product_{context}[create a custom product] and xref:Synchronizing_Repositories_{context}[synchronize repositories].
```

The whole sentence becomes superfluous if we define "you have created a product" as prerequisite for "creating a custom repo".